buildlib/pr/io_demo/az-stage-io-demo.yaml

```diff
@@ -63,6 +65,7 @@ steps:
         --map-by slot \
         --log-dir $(workspace)/${{ parameters.name }} \
         -i ${{ parameters.roce_iface }} \
+        ${{ parameters.extra_env }} \
```

Comment:
parameters.extra_run_args
since it could be other arguments in the future (not just --env)